config/locales/en.yml

```diff
@@ -46,6 +46,9 @@ en:
           attributes:
             tag_ids:
               blank: "Select at least one Tag"
+  admin:
+    calendars:
+      handbook_hint: "The source URL for your calendar feed. See the <a href='https://handbook.placecal.org/reference/supported-calendar-sources'>PlaceCal Handbook</a> if you need help."
```

Comment:
Wondering if it's better to mark html safe here by using `_html` in key. If it's in the render template, then may be forgotten if html is removed from the locale string. No strong feelings but think marginally "safer" to acknowledge the html in the place it is written out so it's obvious what it is, rather than blindly whitelisting whatever ends up here in the template.

Probably more important to follow whatever convention is set in code previously too though.

Reply:
thats a nice convention, i like it